internal/blockchain/ethereum/ethereum.go

```diff
@@ -448,9 +451,9 @@ func (e *Ethereum) eventLoop() {
 				continue
 			}
 
-			// Send the ack - only fails if shutting down
 			if err != nil {
 				l.Errorf("Event loop exiting: %s", err)
```

Comment:
I think it's probably worth something really visible here like:
```suggestion
				l.Errorf("Unable to recover event loop. TERMINATING SERVER: %s", err)
```